service/kinesis/internal/customizations/readtimeout.go

```diff
@@ -0,0 +1,8 @@
+package customizations
```

Comment:
Nit it would be helpful to have a `doc.go` with package godoc explaining this customization and why it is here, github linked issues woudl be awesome too.